packages/core/src/pubsub/pubsub-message.ts

```diff
@@ -96,6 +97,11 @@ export function serialize(message: PubsubMessage): Uint8Array {
         doc: message.stream.toString(),
         stream: message.stream.toString(),
         tip: message.tip.toString(),
+        /*
+          @active-branch
```

Comment:
if your editor is putting these comments in please reconfigure it so it stops doing that.  Or at least make sure to take them out before putting code up for review